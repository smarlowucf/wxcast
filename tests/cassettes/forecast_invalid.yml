interactions:
- request:
    body: null
    headers:
      Connection: [close]
      Host: [nominatim.openstreetmap.org]
      User-Agent: [Python-urllib/3.6]
    method: GET
    uri: https://nominatim.openstreetmap.org/search?q=Fake%2C+FK&format=json
  response:
    body: {string: '[]'}
    headers:
      Access-Control-Allow-Methods: ['OPTIONS,GET']
      Access-Control-Allow-Origin: ['*']
      Connection: [close]
      Content-Type: [application/json; charset=UTF-8]
      Date: ['Fri, 01 Dec 2017 02:35:33 GMT']
      Server: [Apache/2.4.18 (Ubuntu)]
    status: {code: 200, message: OK}
version: 1
