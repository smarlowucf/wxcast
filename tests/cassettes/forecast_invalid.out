Location not found: Fake, FK.
